google-cloud-video-stitcher/samples/acceptance/list_cdn_keys_test.rb

```diff
@@ -21,8 +21,11 @@
     refute_nil cloud_cdn_key
     @cloud_cdn_key_created = true
 
-    assert_output(/CDN keys:\n#{cloud_cdn_key_name}/) do
+    output, = capture_io do
```

Comment:
Huh, I didn't know you could `a, = some_array` to drop the remaining elements. #still_learning_ruby_after_20_years